### DISCUSSION TITLE
Search on Android is not available

### DISCUSSION THREAD

**Discussion by just-ads** [community/community#148051] [Mobile]
The Android search function does not work.
[视频](https://github.com/community/community/discussions/148051#discussioncomment-11683153)

**Comment by ahmad9059** [comment]
Working Check Now I have checked it working fine.

**Reply by just-ads** [reply]
https://github.com/user-attachments/assets/846ea8e8-f21a-4b90-ba40-64ddf24617f3

**Reply by just-ads** [reply]
In the video, I click the search button, and no response is received.

**Comment by ahmad9059** [comment]
I have android I working fine and you can uninstall and reinstall it maybe It can work

**Reply by just-ads** [reply]
I'll try.

**Reply by just-ads** [reply]
Still so.
